ci/pipelines/deploy-to-production.yml

```diff
@@ -736,20 +620,59 @@ jobs:
 
   - name: retag-egress-image-for-test-perf
     plan:
-      - get: pay-ci
-      - get: egress-ecr-registry-prod
+      - in_parallel:
+          steps:
+          - get: egress-ecr-registry-prod
+            params:
+              skip_download: true
+              format: oci
+            trigger: true
+            passed: [smoke-test-egress-on-prod]
+          - get: pay-ci
+      - in_parallel:
+          steps:
+          - task: parse-ecr-release-tag
+            file: pay-ci/ci/tasks/parse-ecr-release-tag.yml
+            input_mapping:
+              ecr-image: egress-ecr-registry-prod
+          - task: parse-perf-release-tag
+            file: pay-ci/ci/tasks/parse-perf-release-tag.yml
+            input_mapping:
+              ecr-repo: egress-ecr-registry-prod
+      - in_parallel:
+          steps:
+          - load_var: release-number
+            file: ecr-release-info/release-number
+          - load_var: perf-tag
+            file: parse-perf-release-tag/tag
+          - task: assume-retag-role
+            file: pay-ci/ci/tasks/assume-role.yml
+            output_mapping:
+              assume-role: assume-retag-role
+            params:
+              AWS_ROLE_ARN: arn:aws:iam::((pay_aws_test_account_id)):role/concourse
+              AWS_ROLE_SESSION_NAME: retag-ecr-image-as-perf
+          - task: generate-docker-creds-config
+            file: pay-ci/ci/tasks/generate-docker-config-file.yml
+            params:
+              USERNAME: ((docker-username))
```

Comment:
Are these docker hub creds? Any reason we need them here and in all other places?

Reply:
Looks like it is to get pay-concourse-runner from docker hub https://github.com/alphagov/pay-ci/blob/215b89995f3224138b311d9aae43cb75be16fb72/ci/tasks/manifest-retag.yml#L7